ui/client/components/graph/EspNode.js

```diff
@@ -149,8 +149,8 @@ joint.shapes.devs.EspNode =  joint.shapes.devs.Model.extend({
   }, joint.shapes.devs.Model.prototype.defaults),
 })
 
-export function getIconHref(node, nodesSettings) {
-  const iconFromConfig = nodesSettings?.[ProcessUtils.findNodeConfigName(node)]?.icon
+export function getIconHref(node, nodeSettings) {
+  const iconFromConfig = nodeSettings?.icon
```

Comment:
Does ?. work in JS?

Reply:
https://babeljs.io/docs/en/next/babel-plugin-proposal-optional-chaining - it works as we have this